snippets/maintainDigits.md

```diff
@@ -0,0 +1,14 @@
+### setDigits
+
+Uses `String.padStart()` or `String.padEnd()` to maintain digit length of a provided integer. Returns stringified integer.
+
+Control direction (0 = before integer, 1 = after integer).
+
+```js
+const setDigits = ( num=0, len=1, direction=0 ) => {
+    return direction > 0 ? num.toString().padStart(len, "0") : num.toString().padEnd(len, "0");
```

Comment:
`padStart` and `padEnd` are not in ES6 but ES2017. That should be noted in the description, even though it is supported in most browsers

Reply:
Should I rewrite it without relying on those instead?

Reply:
Just old mobile & IE11 users can't use pad; they can use a polyfill.